cypress/integration/system_status.js

```diff
@@ -0,0 +1,7 @@
+describe('System Status', () => {
+    it('System Status loaded', () => {
+        cy.get('[data-attr=menu-item-settings]').click()
+        cy.get('[data-attr=menu-item-system-status]').click()
+        cy.get('h1').should('contain', 'System Status')
```

Comment:
I'd add a quick check for something like `cy.get('table').should('contain', 'Postgres Event table')` just to make sure the request completes